app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,11 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def check_rbac?
+    return true if self[:child_id].nil?
+    role_allows?(:feature => self[:child_id])
```

Comment:
Good to know, thx!

In most buttons we will have `@record` and will simply pass that to the `role_allows?` call I guess.